sources/dynamodb/README.md

```diff
@@ -30,28 +30,21 @@ can also pass corresponding source profile connection parameters `aws-access-key
 , `aws-secret-access-key`, `aws-region`. Custom endpoint can be specified using
 `dydb-endpoint` param.
 
-For example, to convert schema run
+For example, to migrate schema run
```

Comment:
Phrase it similar to the other READMEs:

"For example, to perform schema conversion, run"

Reply:
Agreed. But let's be uniform in all the DB specific readme's